server/snapshot/snapshot.go

```diff
@@ -1,11 +1,11 @@
 package snapshot
 
 import (
-	"encoding/json"
-
 	"github.com/Sirupsen/logrus"
 
+	"github.com/docker/go/canonical/json"
```

Comment:
I think this change to canonical json should be fine, but could we test interacting with old repos to ensure there aren't any issues?  cc @endophage since he's more familiar with our canonical/json vs. encoding/json compatibilities